### PR TITLE
Make healthceck conform to new spec. Remove redundant "test" from "te…

### PR DESCRIPTION
**Pull request by chrisbaldauf**
…st_result", key by component, rather than flat array, generated_at instead of report_as_of

### CODE REVIEW

File: api/controllers/admin.js [hunk 1]
```diff
@@ -19,10 +19,13 @@ function healthcheck(req, res) {
   require("../../lib/healthcheck/index.js")()
     .then(function(results){
       // If any tests have failed, return with a 503
-      var failures = results.tests.filter(function(e) {
-        return e.test_result === "failed";
+      var failures = [];
+      Object.keys(results.tests).forEach(function (key) {
```
Comment (chrisbaldauf) [inline api/controllers/admin.js]: I'd be happy to learn of a more idiomatic way to do this.

Comment (ryanbreen) [inline api/controllers/admin.js]: I think you want underscore for `map`: http://underscorejs.org/#map

File: api/controllers/admin.js [hunk 2]
```diff
@@ -19,10 +20,9 @@ function healthcheck(req, res) {
   require("../../lib/healthcheck/index.js")()
     .then(function(results){
       // If any tests have failed, return with a 503
-      var failures = results.tests.filter(function(e) {
-        return e.test_result === "failed";
+      var failures = _.map(results.tests).filter(function(e) {
```
Comment (chrisbaldauf) [inline api/controllers/admin.js:23]: Thanks @ryanbreen!

**Pull request merged**